Modules/Core/Mesh/wrapping/itkTransformMeshFilter.wrap

```diff
@@ -14,7 +14,7 @@ itk_wrap_class("itk::TransformMeshFilter" POINTER)
             "itk::Mesh< ${ITKT_${t}},${d} >, itk::Mesh< ${ITKT_${t}},${d} >, itk::Transform< ${ITKT_${tt}}, ${d}, ${d} >")
       endforeach()
     endforeach()
+    itk_wrap_template("M${ITKM_F}${d}M${ITKM_F}${d}BST${ITKM_D}${d}3"
+          "itk::Mesh< ${ITKT_F},${d} >, itk::Mesh< ${ITKT_F},${d} >, itk::BSplineTransform<${ITKT_D},${d},3>")
```

Comment:
```suggestion
          "itk::Mesh< ${ITKT_F},${d} >, itk::Mesh< ${ITKT_F},${d} >, itk::BSplineTransform<${ITKT_D},${d},${d}>")
```

Reply:
The fist dimension of the transform should match dimension of the input, the second one that of the output mesh. Here we use `${d}` for both.

Reply:
The [B-Spline documentation](https://itk.org/Doxygen/html/classitk_1_1BSplineTransform.html) indicates that only the second of the three template arguments describes mesh dimensions, while the third argument is used to determine mesh size: 
> class itk::BSplineTransform< TParametersValueType, NDimensions, VSplineOrder >
...
> This class encapsulates a deformable transform of points from one N-dimensional space to another N-dimensional space. 
...
> mesh size = number of control points - spline order

In that case shouldn't the third argument be fixed separate from mesh dimensions?

This is reflected in the current wrapping of `BSplineTransform`:
> \>itk.BSplineTransform.GetTypes()
\<itkTemplate itk::BSplineTransform>
Options:
  [<itkCType double>, 2, 3]
  [<itkCType double>, 3, 3]

Reply:
I didn't even look at the docs, sorry about noise.

Reply:
No problem, thanks for checking!